### PR TITLE
Mark cheaters more prominently

### LINKED ISSUE #7809: Mark people who violate academic honesty's profiles more prominently
Currently when I try to visit the certificate of a user who's marked as { "isCheater": true }, instead of showing their certificate, I see "Upon review, this account has been flagged for academic dishonesty."

It would be nice if this message could be featured prominently on that user's profile as well. This would reduce confusion (multiple campers reporting the same user as having cheated, even though they are already flagged as "isCheater" in the database).

### PR DESCRIPTION
**Pull request by karel1980**
Please review. Maybe it's better to do it directly in the view template, but using flash errors freed me from having to think about styling.

closes #7809

### CODE REVIEW

**Comment by karel1980** [comment]
removed keywords and issue number from PR title and commit message.
Because it's now totally obscure which issue this is linked to: it's seven eight oh nine

**Comment by karel1980** [comment]
The camperbot link to the Guidelines for contributing is broken.

**Comment by raisedadead** [comment]
@karel1980 Its here : https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md
Sorry about the broken link. We will fix it. Thanks for pointing out.

File: server/boot/user.js [hunk 1]
```diff
@@ -584,4 +582,14 @@ module.exports = function(app) {
       res.redirect('/map');
     }
   }
+
+  function flashCheater(req, user) {
```
Comment (raisedadead) [inline server/boot/user.js]: CI Fails here
```
/home/travis/build/FreeCodeCamp/FreeCodeCamp/server/boot/user.js
  586:30  error  'user' is defined but never used  no-unused-vars
✖ 1 problem (1 error, 0 warnings)
```

Comment (karel1980) [inline server/boot/user.js]: Ah of course. I was going to add the 'if user.isCheater' inside this function but decided against it.

**Comment by karel1980** [comment]
Updated. I also removed the typo fix, that's fixed in a different branch.

**Comment by DarkAngel6674** [comment]
Please stop sending me these codes and downloads please.  My inbox is to
full so please stop sending them to me. Thank you.
          Tammie Lynn Kennedy
               (830) 480 9732
      darkangel6674@gmail.com

**Comment by karel1980** [comment]
@DarkAngel6674 We don't have control about who receives these messages. You probably subscribed to this (intentionally or by accident) some time ago. You should visit https://github.com/FreeCodeCamp/FreeCodeCamp/pull/7866 and look the 'Unsubscribe' button.

**Comment by raisedadead** [comment]
@DarkAngel6674 

Dear Tammie,
You have accidentally subscribed to all notifications for the FreeCodeCamp repository. 
To stop receiving notifications please visit this repo at https://github.com/FreeCodeCamp/FreeCodeCamp
and click on the Watch/Unwatch tab and set it to not watching.

watcher_picker


Thanks and sorry for your inconvenience.

**Comment by raisedadead** [comment]
@karel1980 Will you add the closes tag in the PR description for the issue this references #7890

**Comment by DarkAngel6674** [comment]
Hi FreeCodeCamp/FreeCod,

I'd like to add you to my professional network on LinkedIn.

- Tammie Lynn

Confirm that you know Tammie Lynn: https://www.linkedin.com/e/v2?e=0-imkjngtd-gb&t=ssuw&tracking=eml-guest-invite-cta&ek=invite_guest&sharedKey=aJiitxFZ&invitationId=6122368791012790272








You received an invitation to connect. LinkedIn will use your email address to make suggestions to our members in features like People You May Know. Unsubscribe here: https://www.linkedin.com/e/v2?e=0-imkjngtd-gb&t=lun&midToken=AQFmrFVhGEd36Q&ek=invite_guest&loid=AQHW_k1HK2HtDAAAAVPcGGVYZTgsFnEe_NiYkhBOkFVTeI6Y6VfTbQ5N2wJ-qyHFWwYLt-bB_ZQIwUrSKk2Q8gHVdeF9UwttAcr8AdTaQX3tp3Of7EZ9SrABU-vLBuHzoF_VO8v5pMwMca1TdKANvc-E-bJI1bHG8yCtRHHzPZIwDnKf4UASI1DAYS2jbce93UyG&eid=0-imkjngtd-gb


This email was sent to reply@reply.github.com.

If you need assistance or have questions, please contact LinkedIn Customer Service: https://www.linkedin.com/e/v2?e=0-imkjngtd-gb&a=customerServiceUrl&ek=invite_guest

&copy; 2016 LinkedIn Corporation, 2029 Stierlin Court, Mountain View CA 94043. LinkedIn and the LinkedIn logo are registered trademarks of LinkedIn.

**Comment by DarkAngel6674** [comment]
Здравствуйте, Yura!

Адрес reply@reply.github.com был добавлен в вашу учётную запись LinkedIn.

Если вы не хотите, чтобы этот адрес был связан с вашей учётной записью, воспользуйтесь указанными ниже инструкциями по удалению адреса.

Введите https://www.linkedin.com/e/v2?e=7p5iry-imkjtalq-w3&amp;t=nas&amp;midToken=AQGn21IFy2uczw&amp;ek=security_add_email_notification в адресную строку браузера.
Войдите, указав ваш текущий адрес эл. почты и пароль.
Найдите «Основной адрес эл. почты» вверху слева и кликните «Изменить/добавить».
Кликните «Удалить» рядом с адресом, который вы хотите удалить.

Благодарим за использование LinkedIn и за помощь в обеспечении безопасности вашей учётной записи!
коллектив компании LinkedIn



&copy; 2016 LinkedIn Ireland Limited. LinkedIn, эмблема LinkedIn и InMail являются товарными знаками компании LinkedIn Corporation в США и/или в других странах. Все права защищены.



Это письмо адресовано участнику Yura Lysenko (Front-end разработчик; разработчик внешнего интерфейса). 
Узнайте, почему это указано по ссылке: https://www.linkedin.com/e/v2?e=7p5iry-imkjtalq-w3&a=customerServiceUrl&midToken=AQGn21IFy2uczw&ek=security_add_email_notification&articleId=4788

Если вам нужна помощь или у вас возникли вопросы, обратитесь в службу поддержки LinkedIn: https://www.linkedin.com/e/v2?e=7p5iry-imkjtalq-w3&a=customerServiceUrl&midToken=AQGn21IFy2uczw&ek=security_add_email_notification

LinkedIn является зарегистрированным фирменным наименованием LinkedIn Ireland Limited.
Зарегистрировано в Ирландии в качестве частной компании с ограниченной ответственностью под номером 477441.
Офис зарегистрирован по адресу: Wilton Plaza, Wilton Place, Dublin 2, Ireland (Ирландия).

**Comment by karel1980** [comment]
@raisedadead The PR description (which is the first comment in this thread IIUC) already contains 'closes #7809'.

**Comment by raisedadead** [comment]
@karel1980 Yeah some one added that, :wink: wasn't me though!

**Comment by raisedadead** [comment]
I tried tested locally. 

I made up two users with email sign-in:
1. raisedadead **<= This username is marked as cheater**
2. mrugeshm

Tests:
1. Not signed in and visit `/raisedadead`  **<= Crashes the app**
2. Signed in with `mrugeshm` and visit `/raisedadead` **<= Doesn't flash the required message**
3. Signed in with `raisedadead` and visit `/raisedadead` **<= Flashes the required message**

@karel1980 Can you confirm this is the behavior?

/cc @QuincyLarson @BerkeleyTrue On a side note I may not have configured the Github OAuth correctly, but that shouldn't matter because I have used email signups (unique) for the test users.

**Comment by raisedadead** [comment]
@karel1980 Do you have updates?

**Comment by raisedadead** [comment]
This PR does not fulfill the user stories that are required.
For scenarios please check [my comment](https://github.com/FreeCodeCamp/FreeCodeCamp/pull/7866#issuecomment-206191255).

Closing as stale as well. Please feel free to address the comments and open a fresh PR.

**Comment by karel1980** [comment]
Sorry, I lost track of this (busy busy). Glad to see the feature has been added via another PR in the mean time.

**Comment by raisedadead** [comment]
Hey @karel1980 
There are lots of other open help wanted issues, please do work on those if you would like. 
Thanks a lot for contributing!

**Pull request closed without merging**